config/preinstall.js

```diff
@@ -0,0 +1,29 @@
+const nodeMajorVersion = process.versions.node.split('.')[0];
+const minNodeMajorVersion = 10;
+const redColor = '\x1b[31m';
+const greenColor = '\x1b[32m';
+const yellowColor = '\x1b[33m';
+
+if (nodeMajorVersion < minNodeMajorVersion) {
+    console.log(redColor);
+    console.log('Node.js version :', process.versions.node);
+    console.error(`Error:\tYour installed Node version is inferior at ${minNodeMajorVersion},
```

Comment:
nit: inferior *to*, or “less than” (sounds better IMO)

Reply:
ok